Studs/UserManager.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  UserManager.swift
+//  Studs
+//
+//  Created by Willy Liu on 2019-03-06.
+//  Copyright © 2019 Studieresan. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+
+final class UserManager {
+
+    static private let disposeBag = DisposeBag()
```

Comment:
Am I blind or where is this used? :P

Reply:
Oh oops, forgot to delete this. Was initially planning to do some more stuff in this class, but had some trouble getting it to work. The disposeBag variable should be deleted.